static/js/compose.js

```diff
@@ -184,13 +185,16 @@ exports.abort_xhr = function () {
 };
 
 exports.zoom_token_callbacks = new Map();
-exports.zoom_xhrs = new Map();
+exports.webex_token_callbacks = new Map();
+exports.video_call_xhrs = new Map();
 
-exports.abort_zoom = function (edit_message_id) {
+exports.abort_video_callbacks = function (edit_message_id) {
     const key = edit_message_id || "";
     exports.zoom_token_callbacks.delete(key);
-    if (exports.zoom_xhrs.has(key)) {
-        exports.zoom_xhrs.get(key).abort();
+    exports.webex_token_callbacks.delete(key);
+    if (exports.video_call_xhrs.has(key)) {
+        exports.video_call_xhrs.get(key).abort();
+        exports.video_call_xhrs.delete(key);
```

Comment:
This refactor to the `video_call_xhrs` variable name would be a great prep refactoring commit.